exporter/awsemfexporter/metric_translator.go

```diff
@@ -31,6 +31,15 @@ const (
 	fieldPrometheusMetricType = "prom_metric_type"
 )
 
+// Prometheus instance availability monitoring metrics - https://prometheus.io/docs/concepts/jobs_instances
+var prometheusInstanceScrapeMetrics = map[string]struct{}{
```

Comment:
Id just make this a configurable feature if the exporter.
Allow specifying `dropFields`.